src/blkio.rs

```diff
@@ -396,7 +396,8 @@ impl<'a> From<&'a Subsystem> for &'a BlkIoController {
                 Subsystem::BlkIo(c) => c,
                 _ => {
                     assert_eq!(1, 0);
-                    ::std::mem::uninitialized()
+                    let v = std::mem::MaybeUninit::uninit();
```

Comment:
I can't say that I like these `MaybeUninit` things. When I did my own pass in the rust agent, I figured out most of them could simply be removed from the code.